src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/UserPasswordResetDetails.java

```diff
@@ -0,0 +1,32 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto;
+
+import ca.corefacility.bioinformatics.irida.model.user.User;
```

Comment:
Completely forgot sorry. Removed the dto though since it's not required but moved the controllers into a login package in 66fd23c47ca7b634ae1918ac77341a2fdf53048e